caffeinated/admin/new/pricing/templates/event_tickets_datetime_attached_tickets_row.template.php

```diff
@@ -13,12 +13,12 @@
  * @var string $add_new_datetime_ticket_help_link
  */
 ?>
-<tr id="advanced-dtt-edit-row-<?php echo $dtt_row; ?>" class="advanced-dtt-edit-row">
+<tr id="advanced-dtt-edit-row-<?php echo esc_attr($dtt_row); ?>" class="advanced-dtt-edit-row">
```

Comment:
```suggestion
<tr id="advanced-dtt-edit-row-<?php echo absint($dtt_row); ?>" class="advanced-dtt-edit-row">
```

you can check the var type at the top of the template (if the template has phpdoc comments)